docs/reference/functions/arithmetic/average.html

```diff
@@ -56,6 +56,14 @@
     simd intrinsics to perform the operation, the scalar results may differ by one unit from
     simd ones and is system dependent.
 
+Supported decorators
+====================================================================================================
+[`derivative`](../decorators.html#derivative)
+:   **Required header: **<script type="preformatted">` #include <eve/function/derivative/average.hpp>`</script>
+
+:   The expression `derivative_1st(average)(x,y)` and `derivative_2nd(average)(x,y)` computes the partial
+    derivatives of $f$, where $f$ is the function $(x,y) \rightarrow \ (x+y)/2$.
```

Comment:
Why change the style of the info ?
I prefer the `The expression `derivative(arg)(x)` computes the derivative of the function at `x`.` style more.
This is valid for all other doc

Reply:
this is the style for more than one parameter
for one it is simpler because there is no ambiguity

Reply:
partial derivatives